libglusterfs/src/inode.c

```diff
@@ -362,6 +363,7 @@ __inode_ctx_free(inode_t *inode)
 static void
 __inode_destroy(inode_t *inode)
```

Comment:
I think we should s/__inode_destroy/inode_destroy/g I thought there was a dead lock.

Reply:
there is no `inode_destroy()`.  Hence will keep this part to later.